disk-buffering/src/main/java/io/opentelemetry/contrib/disk/buffering/internal/storage/responses/ReadableResult.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.disk.buffering.internal.storage.responses;
+
+public enum ReadableResult {
+  SUCCEEDED,
+  CLOSED,
+  FILE_HAS_EXPIRED,
+  PROCESSING_FAILED,
+  NO_CONTENT_AVAILABLE
```

Comment:
this enum value is returned, but no caller reacts on it - so maybe it's effectively the same as processing failed

Reply:
Thanks, it's updated now.